tools/projmgr/include/ProjMgrYamlEmitter.h

```diff
@@ -32,7 +32,8 @@ class ProjMgrYamlEmitter {
    * @return true if executed successfully
   */
   static bool GenerateCbuildIndex(ProjMgrParser& parser,
-    const std::vector<ContextItem*> contexts, const std::string& outputDir);
+    const std::vector<ContextItem*> contexts, const std::string& outputDir,
```

Comment:
Is passing `std::verctor<> contexts` by value intentional? If yes, the `const` doesn't have any effect. I'd rather expect passing by reference to avoid copying.

```suggestion
    const std::vector<ContextItem*>& contexts, const std::string& outputDir,
```